content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -117,12 +121,74 @@ kind: KubeletConfiguration
 address: "192.168.0.8"
 ```
 
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+address: "192.168.0.8"
+```
+Additionally, you can change the slices/lists values of the kubelet configuration. For example, you can override the `clusterDNS` and `featureGates` fields as follows:
+
+Main kubelet configuration file contents:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.9"
+  - "192.168.0.8"
+featureGates:
+  MemoryQoS: true
+```
+
+Contents of a file in `--config-dir` directory:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+clusterDNS:
+  - "192.168.0.2"
```

Comment:
is there a way to clean up the list? Is it a typical requirement?

Reply:
Not sure if I understood your question. Let me take a shot. When you refer to the main config, you are talking about the base kubelet config. The intention behind adding drop-in files is to modify or clean up the values in the base configuration.